Gruntfile.js

```diff
@@ -73,6 +73,10 @@ module.exports = function (grunt) {
           }
         }
       },
+      extensions: {
+        files: ['extensions/local-extensions.js', 'extensions/local-extensions.css'],
+        tasks: ['copy:extensions']
```

Comment:
This task no longer exists.

Reply:
Oh, i think we just need to ensure these are served in dev mode, shouldn't need a copy task.  i can update.